src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Services/AcademicYearDateProvider.cs

```diff
@@ -31,7 +31,7 @@ public DateTime LastAcademicYearFundingPeriod
             get
             {
                 // TODO GET DATE FROM SOURCE
-                return CurrentAcademicYearStartDate.AddDays(79);
+                return new DateTime(_currentDateTime.Now.Year, 10, 19, 18, 0, 0);
```

Comment:
I think this is wrong - on 1st January 2017, the current academic year runs 1/8/16 to 31/7/2017, and the R14 for the previous period is 18/10/2016. This method would return 18/10/2017.

Reply:
Good one now updated